sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_document_model_administration_client.py

```diff
@@ -630,18 +630,8 @@ def callback(raw_response, _, headers):  # pylint: disable=unused-argument
         if classifier_id is None:
             classifier_id = str(uuid.uuid4())
 
-        _doc_types = {}
-        for doc, details in doc_types.items():
-            _source = None
-            if hasattr(details.source, "prefix"):
-                _source = self._generated_models.ClassifierDocumentTypeDetails(
-                    azure_blob_source=self._generated_models.AzureBlobContentSource(
-                        container_url=details.source.container_url, prefix=details.source.prefix))
-            elif hasattr(details.source, "file_list"):
-                _source = self._generated_models.ClassifierDocumentTypeDetails(
-                    azure_blob_file_list_source=self._generated_models.AzureBlobFileListContentSource(
-                        container_url=details.source.container_url, file_list=details.source.file_list))
-            _doc_types[doc] = _source
+        _doc_types = {doc: details._to_generated() for doc, details in doc_types.items()}
```

Comment:
do we need for async too, or is this PR just to show some of the changes?

Reply:
I was focusing on showing the changes, but I'm adding it in the async client too since I'd like to merge this so it shows up on the main auto-generated APIView.